src/exercises/day-2/strings-iterators.md

```diff
@@ -10,10 +10,8 @@ pass. Try avoiding allocating a `Vec` for your intermediate results:
 
 
 ```rust
-// TODO: remove this when you're done with your implementation.
-#![allow(unused_variables, dead_code)]
-
-{{#include strings-iterators.rs:prefix_matches}}
+pub fn prefix_matches(prefix: &str, request_path: &str) -> bool {
```

Comment:
Doing this makes us lose the coupling with the function signature in `strings-iterators.rs`. Could you put the include back so that we can keep the connection?

Reply:
Sure, does it make sense to add a new function to the original string-iterators.rs file and refer to it in this file or just add the line back in as is? The commit I just made does the former.

Reply:
The latter is better: that way we avoid duplication in the solution file and we avoid showing `prefix_matches_unimplemented` to the students (there is no reason for them to see such a function name).

My goal is to reuse as much as possible from the solution file. We compile the code there in CI  so we know it works — when we reuse it, we also have some confidence that the unsolved version is aligned with the solution.